packages/frontend/src/providers/MantineProvider.tsx

```diff
@@ -9,6 +9,14 @@ const themeOverride: MantineThemeOverride = {
     black: Colors.DARK_GRAY1,
     white: Colors.WHITE,
 
+    spacing: {
+        xs: 8,
+        sm: 12,
+        md: 16,
+        lg: 20,
+        xl: 24,
+    },
```

Comment:
is there no way to add custom spacings?

Reply:
yep, there is, but I need to sort typescript overrides. that will be a separate PR